bluemix/terminal/table_test.go

```diff
@@ -38,7 +38,7 @@ func TestEmptyHeaderTable(t *testing.T) {
 	testTable.Add("row1", "row2")
 	testTable.Print()
 	assert.Contains(t, buf.String(), "row1")
-	assert.Equal(t, "       \nrow1   row2\n", buf.String())
+	assert.Equal(t, "\nrow1   row2\n", buf.String())
```

Comment:
**Callout**: Removing the tab space does not change the output visually. 

![image](https://github.com/user-attachments/assets/be6bbc1d-685c-4be8-b709-a3bf677649c9)